Products/ZenModel/migrate/addEmergencyShutdownLevel.py

```diff
@@ -0,0 +1,55 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2016, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+import logging
+log = logging.getLogger("zen.migrate")
+
+import Migrate
+import servicemigration as sm
+sm.require("1.1.5")
+
+from levels import levels
```

Comment:
And, if we are renaming things, maybe this should be "addServiceRunLevels" or "addStartAndEmergencyStopLevels" or similar.